src/cost_estimates/mod.rs

```diff
@@ -103,7 +107,8 @@ pub fn estimate_fee_rate<CE: CostEstimator + ?Sized, CM: CostMetric + ?Sized>(
     metric: &CM,
 ) -> Result<f64, EstimatorError> {
     let cost_estimate = estimator.estimate_cost(&tx.payload)?;
-    let metric_estimate = metric.from_cost_and_len(&cost_estimate, tx.tx_len());
+    // DO NOT SUBMIT: fix this
```

Comment:
Please fix this before merging.  Also, as a general practical rule, please don't submit PRs for review if they have stuff like this in them.

Reply:
This one is still TODO.

Reply:
This has also been fixed by at least 340985579